packages/backup/composer.json

```diff
@@ -0,0 +1,17 @@
+{
+	"name": "automattic/jetpack-backup",
+	"description": "Tools to assist with backing up Jetpack sites.",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"require": {},
+	"autoload": {
+		"files": [
+			"actions.php"
```

Comment:
Would this file be always autoloaded?
Would't we have conflict here? where we are defining the same function multiple times 
if this package were to be included in multiple plugins?

Reply:
You are right. I have removed the one and only function from actions.php, as per your other suggestion. :)